backend/fms_core/template_paths.py

```diff
@@ -17,7 +17,7 @@
     "PROJECT_LINK_SAMPLES_TEMPLATE",
 ]
 
-CONTAINER_CREATION_TEMPLATE = static("submission_templates/Container_creation_v3_4_0.xlsx")
+CONTAINER_CREATION_TEMPLATE = static("submission_templates/Container_creation_v3_5_0.xlsx")
```

Comment:
Minor thing here: for the T.I. branch, I renamed all the templates to _vtest.xlsx so we have a quick way of identifying which one will go with the T.I changes.

Reply:
Ok i'll rename the file to in line with the rest.